docs/architecture/blazor-for-web-forms-developers/migration.md

```diff
@@ -8,8 +8,104 @@ ms.date: 09/11/2019
+
+When using this tool, it is also important to only submit user code; third-party dependencies will be addressed at a later stage.
+
+## Convert project files to new SDK style
+
+The next step in converting a code base to support .NET Core is to ready the project files themselves. A new, leaner style of `.csproj` and `.vbproj` is supported for .NET Core that removes much of the extraneous information that was required but proved difficult to maintain. Some of the benefits of this approach are:
+
+1. Source code is implicitly included in the project (with easy opt-out if needed)
+1. NuGet packages are referenced in the project file and their dependencies are automatically brought in without needing to explicitly list them
+1. Globbing is better supported for including/excluded files
+1. AssemblyInfo properties can be controlled by the project file, which is helpful for build-time setting of versions, copyright, and other assembly level attributes.
+
+A basic `csproj` file will now look like the following code:
+
+```xml
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net46</TargetFramework>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Newtonsoft.Json" Version="11.0.1" />
+  </ItemGroup>
+</Project>
+```
+
+For more information on the SDK style of project, see the [documentation](https://docs.microsoft.com/dotnet/core/tools/csproj).
+
+## Update all dependencies to latest versions
+
+One of the most difficult parts of moving to .NET Core is ensuring that third-party dependencies will continue to work once ported. Since often the source code for these libraries is not available, you are not able to rebuild to target .NET Standard. Many dependencies, however, have been ported to support .NET Standard. By updating to the latest version, you are most likely to have a supported package.
+
+In cases where .NET Standard is not supported, there is also the option of being able to run libraries compiled for .NET Framework on .NET Core. Starting with .NET Standard 2.0, a .NET Framework compatibility mode was introduced that allows libraries compiled against the .NET Framework to run on .NET Core. These libraries will not magically find APIs that don't exist, so there is the chance that they will fail at runtime; thus, targeted testing of the scenarios these libraries are used for is recommended. A warning will appear at build time that there were restored for .NET Framework. Although this scenario may cause some errors, it may also help unblock some porting scenarios.
+
+For more information on supporting third-party dependencies, see the [documentation](https://docs.microsoft.com/dotnet/core/porting/third-party-deps).
+
+## Update project to run on .NET 4.7.2 or higher
```

Comment:
Is this section really relevant when migrating a Web Forms app to .NET Core and Blazor? You're not going to be running on .NET Framework, so the content on quirks doesn't seem applicable.